README.md

```diff
@@ -1,19 +1,29 @@
+This repository contains a collection of installers and examples for deploying, operating and maintaining Postgres clusters using PGO, the Postgres Operator from Crunchy Data as part of [Crunchy Postgres for Kubernetes](https://www.crunchydata.com/products/crunchy-postgresql-for-kubernetes).
 
-For general questions or community support, we welcome you to join our [community Discord](https://discord.gg/BnsMEeaPBV). If you believe you have discovered a bug, please open an issue in the [PGO project](https://github.com/CrunchyData/postgres-operator).
+The use of these examples with PGO and other container images (aside from those provided by Crunchy Data) will require modifications of the examples.
 
-The examples are grouped by various tools that can be used to deploy them.
+### Using these Examples
 
+The examples are grouped by various tools that can be used to deploy them.
+Each of the examples has its own README that guides you through the process of deploying it.
 The best way to get started is to fork this repository and experiment with the examples.
+The examples as provided are designed for the use of PGO along with Crunchy Data's Postgres distribution, Crunchy Postgres, as Crunchy Postgres for Kubernetes.  For more information on the use of container images downloaded from the Crunchy Data Developer Portal or other third party sources, please see 'License and Terms' below.
 
-Each of the examples has its own README that guides you through the process of deploying it.
+### Help with the Examples
+
+* For general questions or community support, we welcome you to join our [community Discord](https://discord.gg/BnsMEeaPBV).
+* If you believe you have discovered a bug, please open an issue in the [PGO project](https://github.com/CrunchyData/postgres-operator).
+* You can find the full Crunchy Postgres for Kubernetes documentation [here](https://access.crunchydata.com/documentation/postgres-operator/v5/).
+* You can find out more information about PGO, the Postgres Operator from [Crunchy Data](https://www.crunchydata.com) at the [project page](https://github.com/CrunchyData/postgres-operator).
+
+### FAQs, License and Terms
 
-You can find the full [PGO documentation](https://access.crunchydata.com/documentation/postgres-operator/v5/) for the project here:
+For more information regarding PGO, the Postgres Operator project from Crunchy Data, and Crunchy Postgres for Kubernetes, please see the [frequently asked questions](https://access.crunchydata.com/documentation/postgres-operator/latest/faq).
 
-[https://access.crunchydata.com/documentation/postgres-operator/v5/](https://access.crunchydata.com/documentation/postgres-operator/v5/)
+For information regarding the software versions of the components included in and Kubernetes version compatibility, please see the [components and capabilities section of the Crunchy Postgres for Kubernetes documentation](https://access.crunchydata.com/documentation/postgres-operator/latest/references/components).
```

Comment:
"Components and Capabilities" should be "Components and _Compatibility_".

https://access.crunchydata.com/documentation/postgres-operator/latest/references/components